en/20/04.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Working with Tuples
+actions: ['checkAnswer', 'hints']
+material:
+  editor:
+    language: sol
+    startingCode:
+      "PriceConsumerV3.sol": |
+        pragma solidity ^0.6.7;
+
+        import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
+
+        contract PriceConsumerV3 {
+          AggregatorV3Interface public priceFeed;
+
+          constructor() public {
+            priceFeed = AggregatorV3Interface(0x8A753747A1Fa494EC906cE90E9f37563A8AF630e);
+          }
+        }
+    answer: |
+      pragma solidity ^0.6.7;
+
+      import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
+
+      contract PriceConsumerV3 {
+        AggregatorV3Interface public priceFeed;
+
+        constructor() public {
+          priceFeed = AggregatorV3Interface(0x8A753747A1Fa494EC906cE90E9f37563A8AF630e);
+        }
+
+        function getLatestPrice() public view returns (int) {
+          (,int price,,,) = priceFeed.latestRoundData();
+          return price;
+        }
+      }
+---
+
+Now... all we have to do is call the function on the contract called `latestRoundData` which has all the information we are looking for, plus some!
+
+- `roundId`: The round ID. Each price update gets a unique round ID. 
```

Comment:
@PatrickAlphaC Can briefly explain the concept of rounds? One or two short sentences should be enough. Thanks!

Reply:
Yes! 

I've updated in a future commit:
```
 The round ID. Each price update gets a unique round ID. Whenever a group of nodes responds with a price update, that specific price update is considered a "round", this way you can say "the price was X during round Y". 
```